flink-staging/flink-hcatalog/pom.xml

```diff
@@ -34,17 +34,64 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<repositories>
+		<repository>
+			<id>cloudera</id>
+			<url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
+		</repository>
+	</repositories>
+
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-java</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-scala</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.hive.hcatalog</groupId>
+			<artifactId>hive-hcatalog-core</artifactId>
+			<version>1.1.0-cdh5.4.0</version>
```

Comment:
This is a known issue with hcatalog, the maven artifact is compiled against hadoop1 and blocks unit testing. 
https://issues.apache.org/jira/browse/HIVE-7349
@fhueske 
I found there is no pre-exist test when I begin to work on this issue. How did hcatalog get tested before? If we stick to the vanilla hcatalog, I guess one way is to move it to hadoop1 profile(flink compiled against the maven hcatalog jar can't be used in a hadoop2 env anyway), which will limit its usage to a large extend since almost all hive production servers on run on hadoop2 now.